types/budgets.d.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+    module LH {
+        module Budgets {
+            export interface Json {
```

Comment:
is there a doc on the budgets JSON schema we can link to or something? I faintly remember reading one at one point, but I could be wrong :)

if not, your comment in the issue might suffice just to have a place for readers to go to understand the format better https://github.com/GoogleChrome/lighthouse/issues/6053#issuecomment-428385930

Reply:
I added a link here and in budgets.js - but they're just placeholders until I write up something better. (There's also an article on the way.)